packages/browser-wallet/src/channel-wallet.ts

```diff
@@ -34,12 +31,11 @@ import {
 import {ADD_LOGS, WALLET_VERSION} from './config';
 import {logger} from './logger';
 import {ChainWatcher} from './chain';
-// TODO: factor UI out of channel-wallet
-// import {Wallet as WalletUi} from './ui/wallet';
 
+export type AnyInterpreter = Interpreter<any, any, any>;
```

Comment:
I suppose that when we care, we should leverage generic types to make `renderUI` easier to write.